src/main/java/pages/BasePage.java

```diff
@@ -0,0 +1,12 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+    protected WebDriver driver;
+    public BasePage(WebDriver driver){
+        this.driver = driver;
+        PageFactory.initElements(driver,this);
```

Comment:
It should be more readable if you put an empty space between arguments

Reply:
done